core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -3998,6 +4264,112 @@ private void populateDenseIndexerList() {
         indexCount = denseIndexers.size();
     }
 
+    private boolean prepareDetachedPartition(long partitionTimestamp, long partitionSize, Path detachedPath, int detachedPartitionRoot) {
```

Comment:
confusing terminology: table writer has opposed workflows, "detach" and "attach". This method is used by "attach" workflow, but it references "detached". Better name would be "preparePartitionForAttach"

Reply:
renamed to `preparePartitionForAttach`